content/docs/zero/upgrading.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Upgrading
+description: This page discusses how to upgrade your cluster to use the latest version of Pomerium Zero.
+keywords:
+  [
+    pomerium zero release,
+    zero release,
+    zero releases,
+    pomerium zero releases,
+    upgrade zero,
+    zero version,
+    latest zero version,
+  ]
+pagination_prev: null
+pagination_next: null
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Upgrade Pomerium Zero
+
+This page shows you how to upgrade your cluster to use the latest version of Pomerium Zero. Refer to the steps for your deployment environment.
+
+:::note
+
+Review the latest [**Core Upgrade guide**](/docs/deploy/core/upgrading) for release details before you upgrade Pomerium Zero.
+
+:::
+
+## How to upgrade Pomerium Zero
+
+<Tabs>
+<TabItem label="Docker" value="docker">
+
+To update Pomerium Zero in Docker:
+
+1. In your `docker-compose.yaml` file, update the Pomerium Zero image tag to specify the [latest tagged release](https://github.com/pomerium/pomerium/tags):
```

Comment:
I think the onboarding instructions say to save the configuration just as `compose.yaml`? Might want to double check the file name for consistency.